### ISSUE TITLE
Action Bar features suggestion

### ISSUE THREAD

**Issue by Olinus10** [Crendgrim/AutoHUD#111]
Would be awesome if the action bar section had similar features like the hotbar, to move down when there's no change to the text in the action bar

**Comment by Crendgrim** [comment]
To clarify, what is the use case for this? Do some servers constantly display fixed text in the action bar you'd like to hide?

**Comment by Olinus10** [comment]
> To clarify, what is the use case for this? Do some servers constantly display fixed text in the action bar you'd like to hide?

yes, fixed or constant, but sometimes just varying stuff, like displaying coordinates or stuff like that
